jcommune-plugins/questions-n-answers-plugin/src/test/java/org/jtalks/jcommune/plugin/questionsandanswers/controller/QuestionsAndAnswersControllerTest.java

```diff
@@ -93,6 +95,7 @@
     public void init() {
         initMocks(this);
         when(controller.getPluginBranchService()).thenReturn(branchService);
+        when(controller.getPluginTopicDraftService()).thenReturn(topicDraftService);
```

Comment:
Eh, will need to get rid of these terrible service tests at some point and replace them with Component Tests.